migration/v0.16.x-v0.17.x/upgrade-apps.md

```diff
@@ -17,6 +17,8 @@
 
 1. Verify if the script added all your custom static clients under `dex.additionalStaticClients`. If so, delete the backup at `${CK8S_CONFIG_PATH}/secrets.yaml.bak`.
 
+1. Restart the blackbox-prometheus-blackbox-exporter pod to make changes active.
```

Comment:
This is ok. But in general I'd like to also have a small snippet that one can copy and run.